stories/bags/Hooks.stories.js

```diff
@@ -1,143 +0,0 @@
-            { id: getCount(), name: "test app 3", type: APP_TYPE },
-            { id: getCount(), name: "test app 4", type: APP_TYPE },
-            { id: getCount(), name: "test app 5", type: APP_TYPE },
-            { id: getCount(), name: "test app 6", type: APP_TYPE },
-            { id: getCount(), name: "test app 7", type: APP_TYPE },
-            { id: getCount(), name: "test app 8", type: APP_TYPE },
-            { id: getCount(), name: "test app 9", type: APP_TYPE },
-            { id: getCount(), name: "test app 10", type: APP_TYPE },
-            { id: getCount(), name: "test app 11", type: APP_TYPE },
-        ],
-    },
-};
-
-const data = { ...originalData };
-
-export const TestAddAndDelete = () => {
-    const addItem = facade.useBagAddItem();
-    const deleteAllItems = facade.useBagRemoveItems();
-
-    const deleteAllHandler = () => {
-        data.contents.items = [];
-        return [200, data];
-    };
-
-    const addItemHandler = (config) => {
-        const newID = getCount();
-        data.contents.items = [
-            ...data.contents.items,
-            { id: newID, name: `added item ${newID}`, type: ANALYSIS_TYPE },
-        ];
-        return [200, data];
-    };
-
-    mockAxios.onGet("/api/bags/default").reply(200, data);
-    mockAxios.onPost("/api/bags/default").reply(addItemHandler);
-    mockAxios.onDelete("/api/bags/default").reply(deleteAllHandler);
-
-    return (
-        <Paper>
-            <Button
-                variant="outlined"
-                onClick={() =>
-                    addItem({
-                        name: "test file 1",
-                        path: "/test/path/1",
-                        type: FILE_TYPE,
-                    })
-                }
-                style={{ margin: 1 }}
-            >
-                Add Item
-            </Button>
-            <Button
-                variant="outlined"
-                onClick={deleteAllItems}
-                style={{ margin: 1 }}
-            >
-                Delete All Items
-            </Button>
-            <Bag />
-        </Paper>
-    );
-};
-
-const deleteData = { ...originalData };
-
-export const TestDelete = () => {
-    const deleteItem = facade.useBagRemoveItem();
-
-    const deleteItemHandler = () => {
-        deleteData.contents.items = [
-            ...deleteData.contents.items.slice(
-                0,
-                deleteData.contents.items.length - 1
-            ),
-        ];
-        return [200, deleteData];
-    };
-
-    mockAxios.onGet("/api/bags/default").reply(200, data);
-    mockAxios.onPost("/api/bags/default").reply(deleteItemHandler);
-
-    return (
-        <Paper>
-            <Button
-                variant="outlined"
-                onClick={() => deleteItem({ id: counter - 1 })}
-                style={{ margin: 1 }}
-            >
-                Delete Items
-            </Button>
-            <Bag />
-        </Paper>
-    );
-};
```

Comment:
I delete this story cos I couldn't it work 😩  and these `hooks` are some what tested by the `Bag.stories.js`.